zebra-chain/src/orchard/shielded_data.rs

```diff
@@ -8,15 +8,13 @@ use std::{
 
 use byteorder::{ReadBytesExt, WriteBytesExt};
 use halo2::pasta::pallas;
+use reddsa::{self, orchard::Binding, orchard::SpendAuth, Signature};
```

Comment:
We're currently doing manual tests that print these types, so I'd like to wait for the hex debug changes in `reddsa` to merge and be released:
https://github.com/ZcashFoundation/reddsa/pull/40

I guess that would be version 0.3.1 or 0.4.0:
https://github.com/ZcashFoundation/reddsa/tags

Reply:
Now merged, need to version

Reply:
Published, updated here in https://github.com/ZcashFoundation/zebra/pull/6013/commits/fe38ce718005b72971021db53b3587044ceeb855